README.md

```diff
@@ -3028,7 +3028,7 @@ Other Style Guides
     // bad - raises exception
     const luke = {}
     const leia = {}
-    [luke, leia].forEach(jedi => jedi.father = 'vader')
+    [luke, leia].forEach(rebel => rebel.surname = 'skywalker')
```

Comment:
technically, you just spoiled it - this was just a translation from english to german. (also it's episode V not VII)